### ISSUE TITLE
Artifact verification section - premature?

### ISSUE THREAD

**Issue by hallyn** [opencontainers/distribution-spec#196]
The 'Artifact Verification' Use case says:

```
Before proceeding to download the individual layers, the engine verifies the manifest's signature, ensuring that the content was produced from a trusted source and no tampering has occurred
```

There currently is no supported way to do that, right?  Or is there an ad-hoc in-line signature approach which some are using?

**Comment by jdolitsky** [comment]
I think some of the use case sections need to be cleaned or removed, they are an artifact from the old spec that do not reflect the state of things

**Comment by jdolitsky** [comment]
Closing for now - this is just one component of issue #60 that will be addressed for v1.0.0-rc2
